### DISCUSSION TITLE
reactiveSearchv3 Sorting by dates

### DISCUSSION THREAD

**Discussion by ilrein** [appbaseio/reactivesearch#1648] [Q&A]
Hello, I have a sample query here:
```
				.reactiveSearchv3([
					{
						id: 'from_date',
						type: 'range',
						dataField: ['scheduled_at'],
						// sortBy: 'asc',
						// size: 20,
						value: {
							start: "now",
							end: "now+24h/d"
						},
					},
				]);
```

I'm trying to pull objects based on 'scheduled_at'. I'd like to try to use the reactiveSearchv3 API to do so, what I want is:

1. Get all docs from now until the end of the day
2. Sort them from earliest to latest

Can't seem to accomplish this. Any help?

**Comment by siddharthlatest** [comment]
@ilrein Your usage here looks right to me. Since its using ReactiveSearch API, we can better help if you can share a code snippet at support@appbase.io.
